source/patterns/@aws-solutions-constructs/aws-eventbridge-sqs/lib/index.ts

```diff
@@ -163,7 +181,5 @@ export class EventbridgeToSqs extends Construct {
       this.sqsQueue.grantPurge(new ServicePrincipal('events.amazonaws.com'));
     }
 
-    // Policy for event to be able to send messages to the queue and Grant Event Bridge service access to the SQS queue encryption key
```

Comment:
What's allowing events to be sent to the queue now without this in there anymore? (Mostly curious but wanted to point it out just in case too)

Reply:
There's a new aws-event-targets library since we first wrote this, the SqsQueue target object adds the permissions automatically. It doesn't add Purge however.